### DISCUSSION TITLE
中俄掌握AI技術 英情報頭子示警：10年內地緣政治格局恐被改變

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#181] [General]
https://news.ltn.com.tw/news/world/breakingnews/3753271
報導說，美國、英國、俄羅斯和中國等世界各國的情報機關，都在努力因應科技的巨大進展，這些科技進展不斷挑戰著數千年來由人類主導的傳統情報行動。

**Comment by cobb110202008** [comment]
現代科技的突破，著實逐漸改變整個政治格局，誰能掌握更尖端的技術就能主導世界的走向。

**Comment by briankuo123** [comment]
想到每次一出現新的科技，這些情報機關就得從頭學習並運用就覺得他們好可憐。

**Comment by wh20502** [comment]
也許這些情報單位可以考慮使用AI來學習或是獲取情報，讓他們訓練成可以分析一些很難懂的情報。

**Comment by 110409538** [comment]
以前需要破譯密碼機，才能知道情報的內容。我認為現在只是用科技代替人腦進行分析。也許以後的間諜是以竊取科技為主流。

**Comment by Shiba010** [comment]
原來AI也可以應用在情報層面上，太可怕了！

**Comment by ghost** [comment]
只希望不要帶來破壞 世界能和平
